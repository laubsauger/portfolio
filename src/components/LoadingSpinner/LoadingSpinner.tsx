import React from 'react';

type Props = {
  size?: string
}

const LoadingSpinner = (props:Props) => {
  const { size } = props;

  return (
    <div className="w-full text-center">
      {/*<Spinner animation="border" style={{ width: '5rem', height: '5rem' }} />*/}
      Loading...
    </div>
  )
};

export default LoadingSpinner;