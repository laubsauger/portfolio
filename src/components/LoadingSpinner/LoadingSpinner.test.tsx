import * as React from 'react';
import LoadingSpinner from './LoadingSpinner';
import {render} from "@testing-library/react";

describe('LoadingSpinner', () => {
  it('renders', () => {
    const { asFragment } = render(<LoadingSpinner />);
    expect(asFragment()).toMatchSnapshot();
  });
});