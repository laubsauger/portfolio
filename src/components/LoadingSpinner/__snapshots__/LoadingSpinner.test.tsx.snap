// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSpinner renders 1`] = `
<DocumentFragment>
  <div
    class="w-full text-center"
  >
    Loading...
  </div>
</DocumentFragment>
`;
